gulpfile.js

```diff
@@ -0,0 +1,51 @@
+const fs = require('fs');
+const captureWebsite = require('capture-website');
+
+
+const themesDir = __dirname + '/themes';
+const screenshotDir = __dirname + '/screenshots';
```

Comment:
I think we should prefer `path.join` for these because Windows.

Reply:
I use Windows and it works but sure.